cpu/msp430-common/panic.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2014 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     msp430
+ * @{
+ *
+ * @file
+ * @brief       Crash handling functions implementation for MSP430 MCUs
+ *
+ * @author      Kévin Roussel <Kevin.Roussel@inria.fr>
+ * @author      Oliver Hahm <oliver.hahm@inria.fr>
+ */
+
+#include "cpu.h"
+#include "lpm.h"
+
+void panic_arch(void)
+{
+    /* disable watchdog and all possible sources of interrupts */
```

Comment:
I'm not sure. From the datasheet I cannot see how this should affect the interrupts. However, should we change this in this PR?